mobile/src/libs/components/header-title/header-title.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+import { styles } from './styles';
+
+type Properties = {
+  hasBadge?: boolean;
```

Comment:
Let's pass badge value (as string) 
```suggestion
  badge?: string;
 // ...
const hasBadge = Boolean(badge);
// ...
{hasBadge && <View /* ...props */ />}
```